test/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/Transform.test.cpp

```diff
@@ -0,0 +1,331 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Core/Containers/Array.hpp>
+#include <OpenSpaceToolkit/Core/Containers/Table.hpp>
+#include <OpenSpaceToolkit/Core/Containers/Tuple.hpp>
+#include <OpenSpaceToolkit/Core/FileSystem/File.hpp>
+#include <OpenSpaceToolkit/Core/FileSystem/Path.hpp>
+#include <OpenSpaceToolkit/Core/Types/Real.hpp>
+
+#include <OpenSpaceToolkit/Mathematics/Geometry/3D/Transformations/Rotations/RotationMatrix.hpp>
+#include <OpenSpaceToolkit/Mathematics/Geometry/3D/Transformations/Rotations/RotationVector.hpp>
+
+#include <OpenSpaceToolkit/Physics/Coordinate/Frame.hpp>
+#include <OpenSpaceToolkit/Physics/Environment/Objects/CelestialBodies/Earth.hpp>
+#include <OpenSpaceToolkit/Physics/Units/Derived/Angle.hpp>
+#include <OpenSpaceToolkit/Physics/Units/Length.hpp>
+
+#include <Global.test.hpp>
+
+TEST(OpenSpaceToolkit_Physics_Coordinate_Frame_Providers_Transform, Test)
+{
+    namespace iau = ostk::physics::coord::frame::providers::iau;
+
+    using ostk::core::types::Shared;
+    using ostk::core::types::Real;
+    using ostk::core::types::String;
+    using ostk::core::ctnr::Tuple;
+    using ostk::core::ctnr::Array;
+    using ostk::core::ctnr::Table;
+    using ostk::core::fs::Path;
+    using ostk::core::fs::File;
+
+    using ostk::math::obj::Vector3d;
+    using ostk::math::geom::d3::trf::rot::Quaternion;
+    using ostk::math::geom::d3::trf::rot::RotationVector;
+    using ostk::math::geom::d3::trf::rot::RotationMatrix;
+
+    using ostk::physics::units::Length;
+    using ostk::physics::units::Angle;
+    using ostk::physics::time::Scale;
+    using ostk::physics::time::Instant;
+    using ostk::physics::time::DateTime;
+    using ostk::physics::coord::Transform;
+    using ostk::physics::coord::Frame;
+    using ostk::physics::env::obj::celest::Earth;
```

Comment:
Move this at the top, outside of the test function. All test files should now comply with this ;)

Reply:
done!